### ISSUE TITLE
Python 3.12 support

### ISSUE THREAD

**Issue by VelizarVESSELINOV** [googleapis/python-bigquery-sqlalchemy#913]
```
pip3.12 install sqlalchemy-bigquery==1.8.0
ERROR: Ignored the following versions that require a different python version: 1.0.0 Requires-Python >=3.6, <3.10; 1.0.0a1 Requires-Python >=3.6, <3.10; 1.1.0 Requires-Python >=3.6, <3.10; 1.2.0 Requires-Python >=3.6, <3.10; 1.2.1 Requires-Python >=3.6, <3.10; 1.2.2 Requires-Python >=3.6, <3.10; 1.3.0 Requires-Python >=3.6, <3.11; 1.4.0 Requires-Python >=3.6, <3.11; 1.4.1 Requires-Python >=3.6, <3.11; 1.4.2 Requires-Python >=3.6, <3.11; 1.4.3 Requires-Python >=3.6, <3.11; 1.4.4 Requires-Python >=3.6, <3.11; 1.5.0 Requires-Python >=3.7, <3.11; 1.6.0 Requires-Python >=3.7, <3.12; 1.6.1 Requires-Python >=3.7, <3.12; 1.7.0 Requires-Python >=3.7, <3.12; 1.8.0 Requires-Python >=3.7, <3.12
ERROR: Could not find a version that satisfies the requirement sqlalchemy-bigquery==1.8.0 (from versions: 0.0.1.dev0, 0.0.2.dev0, 0.0.4, 0.0.5, 0.0.6, 0.0.7)
ERROR: No matching distribution found for sqlalchemy-bigquery==1.8.0
```

Only  sqlalchemy-bigquery==0.0.7 is "compatible" with Python3.12
```
pip3.12 install sqlalchemy-bigquery
Collecting sqlalchemy-bigquery
  Downloading sqlalchemy_bigquery-0.0.7.tar.gz (5.1 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: sqlalchemy-bigquery
  Building wheel for sqlalchemy-bigquery (pyproject.toml) ... done
  Created wheel for sqlalchemy-bigquery: filename=sqlalchemy_bigquery-0.0.7-py3-none-any.whl size=4830 sha256=ab8b02ce94c8950da6d095e5da85adf98cf1d81e27495793235ee025c48f3ea4
  Stored in directory: /Users/vv/Library/Caches/pip/wheels/8c/da/0a/e9872d044671fdb12363a43ab58f07de31e417fbdeac6cba9b
```

**Comment by igorghi** [comment]
@Linchin Would you be able to get this [PR](https://github.com/googleapis/python-bigquery-sqlalchemy/pull/903) merged into master anytime soon?

**Comment by Linchin** [comment]
Thank you for the reminder, we have released a new version with python 3.12 support this week, along with the rest of handwritten bigquery packages.

**Comment by igorghi** [comment]
Thanks great! Thank you 😃
